app/services/redis_service.py

```
@@ -0,0 +1,100 @@
+            return None
+        
+        try:
+            value = self.redis_client.get(key)
+            if value:
+                # Parse JSON and convert to proper format
+                data = json.loads(value)
+                # If it's a list of dictionaries, return as-is for Pydantic models
+                if isinstance(data, list) and data and isinstance(data[0], dict):
+                    return data
+                return data
+            return None
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)